index.html

```diff
@@ -295,26 +295,55 @@ <h1 class="greetings2">About Me</h1>
     </ul>
   </section>
 
-  <!-- Get In Touch-->
+  
   <section class="getintouch">
     <div class="sup-txt3">
       <p class="description5">
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch with me.
       </p>
     </div>
-    <form  method="post" class="get-in-touch-form">
-      <input class="common" type="text" placeholder="Full name" maxlength="30" name="fullname" required class="Name"/>
-        <!--Creating A line divider-->
-        <div class="line"></div>
-      <input class="common" type="email" placeholder="Email address" name="email" id="user-email" required />
-                <!--Creating A line divider-->
-        <div class="line"></div>
-      <textarea placeholder="Enter text here" id="" class="textArea" maxlength="500" name="comment"
-        required></textarea>
-      <button type="submit" class="btn-get">Get in touch</button>
+    <!-- Get In Touch Form STARTs here-->
+    <form  id="get-in-touch" action="https://formspree.io/f/mnqydzlw" method="post">
+      <!--Define the Full Name input-->
+      <ul class = "ul-form">
+        <li>
+          <input class="common" type="text" 
+                                placeholder="Full name" 
+                                maxlength="30"
+                                name="fullname" 
+                                required 
+                                class="Name"/>
+        </li>
```

Comment:
- [ ] Please follow the same design provided by Figma your design is not the same as Figma's design check below :point_down: 

**Figma Design**
![Screenshot from 2023-02-21 23-12-38](https://user-images.githubusercontent.com/45226848/220429396-b0bf9630-1a75-4455-a6d9-83c01b4369f9.png)

**Your Design**
![Screenshot from 2023-02-21 23-21-54 (1)](https://user-images.githubusercontent.com/45226848/220429453-0caa06f6-62bb-4227-9012-9fb3b1723f2a.png)

Reply:
Thank you for the observation.

Reply:
Good job an making those changes 👍 one more thing to mention for this headline of this section is the color please make sure to use the same ones as in figma design, your  title is black and in figma #172B4D